graylog2-server/src/main/java/org/graylog2/security/ldap/LdapSettingsImpl.java

```diff
@@ -353,14 +353,14 @@ public void setGroupIdAttribute(String groupIdAttribute) {
     }
 
     @Override
-    public String getGroupObjectClass() {
-        final Object o = fields.get(GROUP_OBJECT_CLASS);
+    public String getGroupSearchPattern() {
+        final Object o = fields.get(GROUP_SEARCH_PATTERN);
```

Comment:
Could be simply `return Strings.nullToEmpty((String) fields.get(GROUP_SEARCH_PATTERN));`